docs/en/enterprise-edition/content-collections/application-security/get-started/connect-code-and-build-providers/ci-cd-runs/add-azure-pipelines.adoc

```diff
@@ -0,0 +1,99 @@
+:topic_type: task
+
+[.task]
+== Connect Azure Pipelines
+
+Integrate Prisma Cloud Code Security with Azure Pipelines to seamlessly embed vulnerability detection into your Azure DevOps CI/CD pipelines. This integration enables continuous scanning and detection of your your infrastructure as code (IaC) and code repositories as a part of your CI/CD workflows. Additionally, it automates shift-left actions and provides reporting and tracking on the Prisma Cloud administrative console.
+
+[.procedure]
+
+. Before you begin, xref:../../../../administration/create-access-keys.adoc[generate and copy the Prisma Cloud access key] to enable access to Prisma Cloud. The access key includes a Key ID and Secret.
+. Create or open the `.gitlab-ci.yml` file in your repository for editing, or alternatively, navigate to *CI/CD* > *Editor* in your GitLab project.
+. In *Azure DevOps*, create a new pipeline or select an existing pipeline and select *Edit*. 
++
+An editor for the `azure-pipelines.yml` configuration file is displayed.
++
+image::application-security/az-pipelines-1.png[]
+
+. Add the following steps into your pipeline jobs or stages, based on your current configuration.
++
+[source,yaml]
+----
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.8'
+  displayName: 'Install Python 3.8'
+- script: pip install checkov
+  displayName: 'Install Checkov'
+- script: checkov -d <directory> --bc-api-key <prisma_access_key>::<prisma_secret_key> --repo-id <org/repo> --branch <branch>
+  displayName: 'Scan with Prisma Cloud'
+  env:
+    PRISMA_API_URL: <prisma_stack_api>
+----
+
+The code snippet above includes the following arguments:
+
+* <directory>: The directory of the repository you wish to scan.
+
+* <prisma_access_key>::<prisma_secret_key>: A combination of your Prisma Access Key and Prisma Secret Key. As a best practice, store this access the access key and secret in a vault
+
+* <org/repo>: Your VCS organization name and repository name
+
+* <branch>: The branch to be scanned
+
+* <prisma_stack_api>: The *API URL* for your Prisma Cloud stack. Refer to the xref:../../../../get-started/console-prerequisites.adoc[available list of URLs] for more information. 
+
+NOTE: Use `--soft-fail` to scan the build for errors without failing the job or stage.
+
+For additional scan settings, refer to the https://www.checkov.io/2.Basics/CLI%20Command%20Reference.html[CLI Command Reference guide].
+
+After completing the integration, Prisma Cloud will automatically conduct a scan, and the outcomes will be presented in the pipeline logs and on the *Projects* page of the Prisma Cloud console. 
+
+image::application-security/az-pipelines-2.png[]
```

Comment:
Need to add this image: https://github.com/PaloAltoNetworks/prisma-cloud-docs/blob/maxwell-fixes/code-security/admin_guide/_graphics/az-pipelines-2.png

Reply:
This image seems to be in the document